### ISSUE TITLE
Access forbidden downloading input data

### ISSUE THREAD

**Issue by neil9876** [EarthWorksOrg/EarthWorks#87]
I have successfully built a case of EarthWorks on narval.alliancecan.ca with --res mpasa120_mpasa120 --compset F2000climo, and I am attempting to submit the case, which requires downloading input datasets. I have transferred the EarthWorks input data via Globus. However, there still appear to be some files that I cannot download. The svn commands are returning Access Forbidden, see output attached.

I tried running `svn ls https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata` as instructed, but that prompted me for a username and password for the server, which I don't have.

[log.submit.txt](https://github.com/user-attachments/files/18050611/log.submit.txt)

**Comment by dazlich** [comment]
The file mpasa120.graph.info.part.24 was never generated. I have created one and added it to the Earthworks input data directory.

Sent from my iPhone

On Dec 7, 2024, at 8:20 PM, neil9876 ***@***.***> wrote:

﻿

** Caution: EXTERNAL Sender **

I have successfully built a case of EarthWorks on narval.alliancecan.ca with --res mpasa120_mpasa120 --compset F2000climo, and I am attempting to submit the case, which requires downloading input datasets. I have transferred the EarthWorks input data via Globus. However, there still appear to be some files that I cannot download. The svn commands are returning Access Forbidden, see output attached.

I tried running svn ls https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata as instructed, but that prompted me for a username and password for the server, which I don't have.

log.submit.txt<https://github.com/user-attachments/files/18050611/log.submit.txt>
